docs/en/query_language/functions/geo.md

```diff
@@ -151,4 +151,36 @@ SELECT geohashDecode('ezs42') AS res
 └─────────────────────────────────┘
 ```
 
+## geoToH3
+
+Calculates H3 point index `(lon, lat)` with specified resolution.
+
+```
+geoToH3(lon, lat, resolution)
+```
+
+**Input values**
+
+- `lon` — Longitude. Type: [Float64](../../data_types/float.md).
+- `lat` — Latitude. Type: [Float64](../../data_types/float.md).
+- `resolution` — Index resolution. Range: `[0, 15]`. Type: [UInt8](../../data_types/int_uint.md).
+
+**Returned values**
+
+- Hexagon index number.
+- 0 in case of error.
```

Comment:
what kind of error? and how to understand the reason of error?

Reply:
Made a separate ticket on update of this and finalizeAggregation functions.